desktop/cmp/store/StoreFilterField.js

```diff
@@ -88,6 +88,7 @@ export class StoreFilterField extends Component {
 
         throwIf(props.gridModel && props.store, "Cannot specify both 'gridModel' and 'store' props.");
         throwIf(props.includeFields && props.excludeFields, "Cannot specify both 'includeFields' and 'excludeFields' props.");
+        warnIf(!props.gridModel && !props.store && isEmpty(props.includeFields), "Must specify one of 'gridModel', 'store', or 'includeFields' or the filter will be a no-op");
```

Comment:
Had this as a throw initially, but that would mean you can't just add an empty StoreFilterField, which I often find useful for setting up initial layouts/mocks, so changed to a warning